### PR TITLE
LPS-101421 SF

### PR DESCRIPTION
**Pull request by brunofarache**

### CODE REVIEW

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -60,6 +56,10 @@
 				}
 			}
 		}
+
+		.panel-group {
```
Comment (jeyvison) [inline modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss]: @brunofarache Why did you have to add this?

Comment (brunofarache) [reply modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/css/main.scss]: @jeyvison didn’t add, reordered alphabetically

**Comment by jeyvison** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jeyvison** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79416. [See changes here.](https://github.com/jeyvison/liferay-portal/compare/68e7d01d2549~1...68e7d01d2549) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**